plugins/com.google.cloud.tools.eclipse.projectselector/src/com/google/cloud/tools/eclipse/projectselector/ProjectRepository.java

```diff
@@ -121,7 +122,7 @@ public AppEngine getAppEngineApplication(Credential credential, String projectId
       // just in case the API changes and exception with 404 won't be
       // used to indicate a missing application
       return AppEngine.withId(application.getId());
-    } catch (IOException ex) {
+    } catch (IOException | GoogleApiException ex) {
```

Comment:
Now we're at it, I think it's better to

```java
} catch (GoogleJsonResponseException ex) {
  ...
} catch (IOException | GoogleApiException ex) {
  ...
}
```

and remove the `if (ex instanceof GoogleJsonResponseException)` {.